src/Components/Auth/Login.tsx

```diff
@@ -282,7 +282,7 @@ export const Login = (props: { forgot?: boolean }) => {
                   forgotPassword && "hidden",
                 )}
               >
-                <div className="mb-8 w-[300px] text-4xl font-black text-primary-600">
+                <div className="mb-8 w-[300px] break-all text-4xl font-black text-primary-600">
```

Comment:
@sunny-thakurwar This seems to be breaking in both mobile view and desktop view even for english. It'd be great if you could test the changes out before requesting a review 😄 

<img width="546" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/1a1cdeb5-4b20-4df9-9ca9-9bd1dca89f51">

<img width="756" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/e22bd7b6-2edd-43b5-bd06-d699da884d7c">

Reply:
@rithviknishad   Opps! i didn't checked with english language. srry for that corrected the responsiveness please check now.